kong/plugins/oidc/utils.lua

```diff
@@ -132,4 +133,10 @@ function M.cache_get(type, key)
   return value
 end
 
+function M.clear_request_headers()
+  ngx.req.clear_header("X-Access-Token")
+  ngx.req.clear_header("X-ID-Token")
+  ngx.req.clear_header("X-Userinfo")
```

Comment:
should these be hard coded?  how likely are they to need to be changed / added to in the future?  can we read them from a configurable list instead?

Reply:
Updated to use constants. Chris and I agree these aren't values that will likely every be changed/added to.